### PR TITLE
Add privacy policy

### PR DESCRIPTION
**Pull request by AurelReb**
This PR is related to #246 
The last thing I should do is put the HTML content in a markdown file but I don't know how to do it. Any idea?

### CODE REVIEW

**Comment by AurelReb** [comment]
everything should be good now :D

File: prologin/pages/templates/pages/about-privacy-policy.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+{% breadcrumb_for "pages:about-privacy-policy" %}{% trans "Privacy policy" %}
+{% endbreadcrumb_for %}
+{% endblock breadcrumbs %}
+
+{% block title %}{% trans "Privacy policy" %}{% endblock %}
+
+{% block content %}
+
+<!-- Translation of markdown -->
+{% blocktrans asvar about_processing %}
```
Comment (seirl) [inline prologin/pages/templates/pages/about-privacy-policy.html]: Put each blocktrans above its specific section instead of putting everything here.

**Review by seirl — CHANGES_REQUESTED** [review]
We need to find a way to have a smaller diff for django.po with just the relevant section.

**Comment by AurelReb** [comment]
Indeed, it's an online .po editor that made as much diff. I forgot to re-`makemessage` after translate it to strip the file.

**Comment by AurelReb** [comment]
I replaced the previous commit so the .po file diff is clear now. I also put each blocktrans in their specific section

File: prologin/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1255,7 +1255,7 @@ msgstr "Export données"
 
 #: documents/templates/documents/index.html:115
 msgid "Portrayal agreement Orga"
-msgstr ""
+msgstr "Accord de représentation Orga"
```
Comment (seirl) [inline prologin/locale/fr/LC_MESSAGES/django.po]: Droit à l'image ?

File: prologin/locale/fr/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -1022,7 +1022,7 @@ msgstr "Réponses du QCM"
 #: contest/templates/correction/final.html:14
 #: contest/templates/correction/final.html:22
 msgid "All regional event"
-msgstr ""
+msgstr "Tous les événements régionaux"
```
Comment (seirl) [inline prologin/locale/fr/LC_MESSAGES/django.po]: Épreuves régionales ?

**Review by seirl — APPROVED** [review]

**Pull request merged**